### ISSUE TITLE
Users should be able to non-interactively tell jekyll-s3 what to do about deleted files

### ISSUE THREAD

**Issue by drocamor** [laurilehmijoki/jekyll-s3#8]
We're using jekyll-s3 on a build server to allow designers to easily publish changes. This works well until jekyll-s3 asks for confirmation when a file has been removed. 

For example:

```
foo.html is on S3 but not in your _site directory anymore. Do you want to [d]elete, [D]elete all, [k]eep, [K]eep all?
```

Can these become options at the command line or in the config file so that it's easier to automate jekyll-s3?

Thanks!

**Comment by laurilehmijoki** [comment]
Making the delete confirmation optional is a good idea.

I'm a bit busy at the moment. Maybe you can send a pull request that
contains the implementation, Rspecs and at least one Cucumber test?

Cheers
Lauri

**Comment by laurilehmijoki** [comment]
Hello

I just released the version 0.0.7 of jekyll-s3. The new version has a headless mode, in which the interactive delete is disabled.

Here's more info on the new mode: https://github.com/laurilehmijoki/jekyll-s3#the-headless-mode.

Happy blogging
Lauri

**Comment by drocamor** [comment]
wow thanks!
